csrankings.csv

```diff
@@ -15861,6 +15864,7 @@ Robert D. Stevens,University of Manchester,https://www.research.manchester.ac.uk
 Robert D. Tennent,Queen’s University,http://www.cs.queensu.ca/people/faculty.php,NOSCHOLARPAGE
 Robert D. van der Mei,CWI,http://www.few.vu.nl/~mei,MTrV8cgAAAAJ
 Robert Dabrowski,University of Warsaw,https://www.facebook.com/public/Robert-Dabrowski/school/Warsaw-University-of-Technology-109438579082305,jJXYs6EAAAAJ
+Robert Dyer,University of Nebraska,https://go.unl.edu/~rdyer,-tLiAa8AAAAJ

```

Comment:
Please correct web page (dead link) to http://cse.unl.edu/~rdyer/index.php?full=1.

Reply:
Link isnt dead? Works fine.
![image](https://user-images.githubusercontent.com/1115390/95668892-60371580-0b3f-11eb-8745-72768286ddd4.png)


Also note the link given is a URL shortener, so it will be updated as needed (thus requiring fewer updates to this list).

Reply:
Oh I see the issue now. Typo in the shortened url, not in the resulting url.

Fix pushed.